src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -5,33 +5,45 @@
 	private static final int EMPTY = -1;
 	private static final int NUMBERS_SIZE = 12;
 	private static final int FULL = 11;
-	private int[] numbers = new int[NUMBERS_SIZE];
+	private int[] numbers = new int[getNumbersSize()];
 
-	private int total = EMPTY;
+	private int total = getEMPTY();
 
-	public void countIn(int in) {
+    public static int getEMPTY() {
```

Comment:
Encapsulation concerns mainly non-static attributes.
Encapsulation of constants is a significant exaggeration!